src/components/shared/SignupModal.tsx

```diff
@@ -0,0 +1,46 @@
+interface SignupModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const SignupModal = ({ isOpen, onClose }: SignupModalProps) => {
+  return (
+    <div>
+      <label
+        htmlFor="my-modal-3"
+        className={`modal cursor-pointer ${isOpen && "modal-open"}`}
+      >
+        <div className="modal-box relative">
+          <label
+            htmlFor="my-modal-3"
+            onClick={() => onClose()}
+            className="btn btn-sm btn-circle absolute right-2 top-2 text-black bg-white border-none hover:bg-gray-400"
+          >
+            ✕
+          </label>
+          <h3 className="text-4xl font-bold mx-auto w-fit text-primary">
+            Join <a className="text-secondary">M</a>Clubs
+          </h3>
+          <div className="flex justify-center mt-5 mb-5 font-semibold ">
+            <ul className="list-disc font-inter list-outside">
+              <li className="my-1">Save favorites to your profile</li>
+              <li className="my-1">Add your club to the website</li>
+              <li className="my-1">Write reviews</li>
+              <li className="my-1">Submit questions to clubs</li>
+            </ul>
+          </div>
+          <div className="flex flex-col justify-center items-center gap-4 ">
+            <button className="btn btn-outline max-w-xs w-full text-primary hover:bg-primary font-inter text-base">
+              Login with UMID
+            </button>
+            <button className="btn bg-primary text-white max-w-xs w-full hover:bg-white hover:text-primary font-inter text-base">
+              Create Account with UMID

```

Comment:
```suggestion
              Create an Account with your UM email
```
I'd reword this. If we say "UMID" the user might think it's through the UM weblogin, which isn't the case. I'm guessing you guys followed the designs, so we gotta sync with the designers tomorrow.